ballerina/Client.bal

```diff
@@ -148,20 +138,15 @@ type ClientConfiguration record {|
     sql:ConnectionPool? connectionPool;
 |};
 
-# MSSQL database options.
+# Provides a set of configuration related to MSSQL database.
 #
 # + secureSocket - SSL Configuration to be used
-# + socketTimeout - The number of milliseconds to wait before a timeout occurs
-#                   on a socket read or accept. The default value is 0, which means 
-#                   infinite timeout
-# + queryTimeout - The number of seconds to wait before a timeout has occurred on a 
-#                  query. The default value is -1, which means infinite timeout.
-#                  Setting this to 0 also implies to wait indefinitely
-# + loginTimeout - The number of seconds the driver should wait before timing out a 
-#                  failed connection. A zero value indicates that the timeout is the
-#                  default system timeout, which is specified as 15 seconds by default.
-#                  A non-zero value is the number of seconds the driver should wait
-#                  before timing out a failed connection
+# + socketTimeout - Socket timeout (in seconds) during the read/write operations with the MSSQL server
+#                   (0 means no socket timeout)
+# + queryTimeout - Timeout (in seconds) to be used when executing a query.
+#                  (-1/0 means no query timeout)
+# + loginTimeout - Timeout (in seconds) when connecting to the MSSQL server and authentication.
+#                  (0 means 15s of login timeout which is the default behaviour of the driver)
```

Comment:
if the login timeout is configured as 0 by a user, Is it correct to set default as 15s

Reply:
This is actually a bug and needs to be fixed. I was planning to send the fix seperately.

Reply:
Ack. Better we do not mention it in API docs. shall we just say? (default timeout is 15s)

Reply:
Fixed in 46adcef